layers/+spacemacs/spacemacs-layouts/README.org

```diff
@@ -5,10 +5,53 @@
 * Table of Contents                     :TOC_5_gh:noexport:
 - [[#description][Description]]
   - [[#features][Features:]]
+- [[#make-funtions-work-within-layouts-buffers-only][Make funtions work within layout's buffers only]]
 
 * Description
 This layer adds support for distinct layouts/workspaces to Spacemacs.
 
 ** Features:
 - Support for distinct layouts via =eyebrowse=
 - Integration with =helm= and =ivy= to search for buffers within layouts
+* Make funtions work within layout's buffers only
+ Besides =helm-min= and =ivy= that are aware of layout's buffers the following
+ Spacemacs functions also by default are adviced to work within layout's buffers
+ only.
+ #+begin_example
+  next-buffer
+  previous-buffer
+  spacemacs/window-split-double-columns
+  spacemacs/window-split-single-column
+  spacemacs/window-split-triple-columns
+  spacemacs/window-split-grid
+ #+end_example
+
+ They are wrapped around by persp-mode ~with-persp-buffer-list~. The variable
+ ~spacemacs-layouts-advice-list~ holds the list of these functions. Basically
+ you can try adding functions that call ~buffer-list~. For example you can add
+ ~helm-do-ag-buffers~ to the list if you want it to search for buffers of the
+ current layout only not all the buffers accross all layouts.
+ 
+ Function ~spacemacs/alternate-buffer(SPC Tab)~ is also made to be work within
+ layouts. If you want the old behavior set ~spacemacs-layouts-restrict-spc-tab~
+ to nil.
+ 
+ To customize, if you don't want any of these changes add this to your
+ ~dotspacemacs/layers~
+ #+begin_example elisp
+ (spacemacs-layouts :variables
+                    spacemacs-layouts-restrict-spc-tab nil
+                    spacemacs-layouts-advice-list '())
+ #+end_example
+ 
+ if you enjoy this feature and want to add ~helm-do-ag-buffers~ to it:
+ #+begin_example elisp
```

Comment:
Move up this example block, below the line:
>If you enjoy this feature and want to add \~helm-do-ag-buffers\~ to it:

In this review it's on line 35, but it might change when the review changes are applied.